libs/clients/auth/public-api/src/lib/authPublicApiClient.module.ts

```diff
@@ -2,9 +2,18 @@ import { Module } from '@nestjs/common'
 
 import { ApiConfiguration } from './apiConfiguration'
 import { exportedApis } from './apis'
+import { AuthPublicApiClientConfig } from './authPublicApiClient.config'
 
+export const AuthPublicApiClientConfigProvider = {
+  provide: AuthPublicApiClientConfig.KEY,
+  useValue: AuthPublicApiClientConfig,
+}
 @Module({
-  providers: [ApiConfiguration, ...exportedApis],
+  providers: [
+    ApiConfiguration,
+    AuthPublicApiClientConfigProvider,
```

Comment:
Why?

Services should be able to inject it if it's been loaded correctly in the application root module.

Reply:
I made the change to get my schemas to build.
No idea why it behaves like it does on my end.
```
[App] Error	7/18/2023, 5:45:50 PM [ExceptionHandler] Nest can't resolve dependencies of the AuthPublicApiClientConfiguration (?). Please make sure that the argument CONFIGURATION(AuthPubliClient) at index [0] is available in the AuthPublicApiClientModule context.

       Potential solutions:
       - If CONFIGURATION(AuthPubliClient) is a provider, is it part of the current AuthPublicApiClientModule?
       - If CONFIGURATION(AuthPubliClient) is exported from a separate @Module, is that module imported within AuthPublicApiClientModule?
         @Module({
           imports: [ /* the Module containing CONFIGURATION(AuthPubliClient) */ ]
         })
```